README.md

```diff
@@ -10,9 +10,12 @@ Ensure that docker is installed on the machine where the tests need to be run.
 
 Then, run the following as usual.
 ```
-cargo test
+DOCKER_BUILD=1 cargo test
```

Comment:
does the value this is set to matter, or does it just need to be set for `env::var("DOCKER_BUILD").is_ok()` to be true?

Reply:
The value doesn't matter. Just that it is set to something. So you could do DOCKER_BUILD=candy and that would work also.